src/pages/components/header/menu/Menu.tsx

```diff
@@ -3,25 +3,46 @@ import { useLocation, useNavigate } from 'react-router-dom'
 import cn from 'classnames'
 import './menu.scss'
 
+import { useEffect, useState } from 'react'
+
 export type MenuPage = {
   label: string
   key: string
 }
 
-function Menu({ pages }: { pages: MenuPage[] }) {
+function Menu({ pages, shouldDismiss }: { pages: MenuPage[]; shouldDismiss: () => void }) {
   const navigate = useNavigate()
   const { pathname: currpage } = useLocation()
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 768)
+
   console.log({ currpage, active: pages.map((p) => p.key === currpage) })
 
+  function navigateTo(page: MenuPage, dismiss: boolean) {
+    page.key[0] === '/' ? navigate(page.key) : window.open(page.key, '_blank')
+    if (dismiss) {
+      shouldDismiss()
```

Comment:
I think the naming convention is confusing
shouldDismiss is the funtion that dismiss the menu, and dismiss is the boolean that tell us whether the menu should be dismissed?

Reply:
I agree, no idea what I was thinking :) the name will be switched in order to make sense